src/it/feature-should-fail-if-empty-report/pom.xml

```diff
@@ -19,6 +19,9 @@
                 <groupId>@project.groupId@</groupId>
                 <artifactId>@project.artifactId@</artifactId>
                 <version>@project.version@</version>
+                <configuration>
+                    <reportVersion>2.0.1</reportVersion>
+                </configuration>
```

Comment:
`[ERROR] The following builds failed:`
`[ERROR] *  feature-should-fail-if-empty-report/pom.xml`
In old schema we did not generate report for empty directory, but now `allure command line` ignores this problem and generate an empty report without any fails. You can fix this behaviour by removing `invoker.properties` file or set `invoker.buildResult` property to `success`.